packages/react-components/react-textarea/stories/Textarea/TextareaAppearance.stories.tsx

```diff
@@ -1,58 +1,47 @@
 import * as React from 'react';
-import { makeStyles, shorthands, tokens, useId, Label, Textarea } from '@fluentui/react-components';
+import { Field, makeStyles, mergeClasses, shorthands, tokens, Textarea } from '@fluentui/react-components';
 
 const useStyles = makeStyles({
   base: {
     display: 'flex',
     flexDirection: 'column',
-
-    '> div': {
-      display: 'flex',
-      flexDirection: 'column',
-      marginTop: tokens.spacingVerticalMNudge,
-      ...shorthands.padding(tokens.spacingHorizontalMNudge),
-    },
-
-    '> div > label': {
-      marginBottom: tokens.spacingHorizontalXXS,
-      marginLeft: tokens.spacingHorizontalMNudge,
-    },
+    rowGap: tokens.spacingVerticalMNudge,
   },
   filledLighter: {
     backgroundColor: tokens.colorNeutralBackgroundInverted,
-    '> label': {
-      color: tokens.colorNeutralForegroundInverted2,
-    },
   },
   filledDarker: {
     backgroundColor: tokens.colorNeutralBackgroundInverted,
-    '> label': {
-      color: tokens.colorNeutralForegroundInverted2,
-    },
+  },
+  labelFilled: {
+    color: tokens.colorNeutralForegroundInverted2,
+  },
+  field: {
+    ...shorthands.padding(tokens.spacingVerticalMNudge, tokens.spacingHorizontalMNudge),
   },
```

Comment:
A few nits:
* The class name probably shouldn't be `field` since it's no longer applied to a Field.
* The `field`, `filledLigher`, and `filledDarker` classes could be combined into a single class like `inverted`.
* Maybe rename `labelFilled` to `invertedLabel`.

```ts
  inverted: {
    backgroundColor: tokens.colorNeutralBackgroundInverted,
    ...shorthands.padding(tokens.spacingVerticalMNudge, tokens.spacingHorizontalMNudge),
  },
  invertedLabel: {
    color: tokens.colorNeutralForegroundInverted2,
  },
```

Reply:
Ah, I see the padding is also applied around the non-inverted example, so maybe `field` still needs to be a separate class, maybe `fieldWrapper` or something.